From 79660b48f9c3ad40aeba865baf7f7d61ef45a9f3 Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Thu, 13 Sep 2018 17:21:02 +0200
Subject: [PATCH] TRITON-762 Remove incremental upgrade scripts from
 sdc-headnode Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent
 Mick <trentm@gmail.com>

---
 Makefile               | 18 +-----------------
 tools/purge-mg-builds  |  1 -
 tools/targets-1.6.3.sh |  4 ++--
 3 files changed, 3 insertions(+), 20 deletions(-)

diff --git a/Makefile b/Makefile
index 4701f4b..416696a 100644
--- a/Makefile
+++ b/Makefile
@@ -2702,8 +2702,7 @@ HEADNODE_TARGETS = \
 # These targets are only built for the base "headnode" job.
 #
 HEADNODE_BASE_ONLY_TARGETS = \
-	gz-tools \
-	incr-upgrade
+	gz-tools
 
 .PHONY: headnode
 headnode: $(HEADNODE_TARGETS) $(HEADNODE_BASE_ONLY_TARGETS)
@@ -2831,21 +2830,6 @@ gz-tools_publish_image:
 	$(UPDATES_IMGADM) import -ddd -m $(GZ_TOOLS_MANIFEST_OUTPUT) -f $(GZ_TOOLS_OUTPUT)
 	@echo ""
 
-INCR_UPGRADE_BUILD=$(USB_BUILD_DIR)/incr-upgrade-$(_headnode_stamp).tgz
-INCR_UPGRADE_OUTPUT=$(USB_BITS_DIR)/incr-upgrade$(HEADNODE_SUFFIX)-$(_headnode_stamp).tgz
-
-.PHONY: incr-upgrade
-incr-upgrade: $(INCR_UPGRADE_OUTPUT)
-
-$(INCR_UPGRADE_OUTPUT): $(USB_BITS_SPEC) $(BOOT_OUTPUT)
-	@echo "# Build incr-upgrade: sdc-headnode branch $(SDC_HEADNODE_BRANCH), sha $(SDC_HEADNODE_SHA), time `date -u +%Y%m%dT%H%M%SZ`"
-	cd "build/sdc-headnode" && \
-	    BITS_DIR=$(BITS_DIR) TIMESTAMP=$(TIMESTAMP) \
-	    gmake incr-upgrade
-	mv $(INCR_UPGRADE_BUILD) $(INCR_UPGRADE_OUTPUT)
-	@echo "# Created incr-upgrade bits (time `date -u +%Y%m%dT%H%M%SZ`):"
-	@ls -l $(INCR_UPGRADE_OUTPUT)
-	@echo ""
 
 #
 # Of the "headnode*" family of builds, only the base "headnode" job currently
diff --git a/tools/purge-mg-builds b/tools/purge-mg-builds
index 55db846..02d5ccf 100755
--- a/tools/purge-mg-builds
+++ b/tools/purge-mg-builds
@@ -77,7 +77,6 @@ export PATH=$TOP/node_modules/.bin:$PATH
 #
 # Deliberately excluded build names for now (and the reason why):
 #    agentsshar-upgrade             paranoia
-#    incr-upgrade                   paranoia
 #    platform                       want to discuss with OS guys
 #    platform-debug                 want to discuss with OS guys
 #    sdcsso                         build story isn't clear, eng hands off here
diff --git a/tools/targets-1.6.3.sh b/tools/targets-1.6.3.sh
index 5109aa4..e5779f2 100755
--- a/tools/targets-1.6.3.sh
+++ b/tools/targets-1.6.3.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -28,7 +28,7 @@ cd ${BASE}
 if [[ ! -f "targets.json" ]]; then
     bash < targets.json.in | json > targets.json
 fi
-node -e "var targets = require('./targets.json'); Object.keys(targets).forEach(function (t) { if (['all', 'usbheadnode', 'usbheadnode-debug', 'platform', 'platform-debug', 'agentsshar', 'minnow', 'amon', 'registar', 'config-agent', 'mockcloud', 'incr-upgrade', 'zonetracker'].indexOf(t) !== -1) { return; }; if (targets[t].image_uuid !== 'b4bdc598-8939-11e3-bea4-8341f6861379') { console.log(t); } });" | sort
+node -e "var targets = require('./targets.json'); Object.keys(targets).forEach(function (t) { if (['all', 'usbheadnode', 'usbheadnode-debug', 'platform', 'platform-debug', 'agentsshar', 'minnow', 'amon', 'registar', 'config-agent', 'mockcloud'].indexOf(t) !== -1) { return; }; if (targets[t].image_uuid !== 'b4bdc598-8939-11e3-bea4-8341f6861379') { console.log(t); } });" | sort
 
 # This comes last as it depends on all the individual agents to be built first.
 echo "agentsshar"
-- 
2.21.0

